File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Config/UnicodeStringEncodingTestCase.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System;
using Db4objects.Db4o.Internal.Encoding;
using Db4objects.Db4o.Tests.Common.Config;

namespace Db4objects.Db4o.Tests.Common.Config
{
	/// <exclude></exclude>
	public class UnicodeStringEncodingTestCase : StringEncodingTestCaseBase
	{
		protected override Type StringIoClass()
		{
			return typeof(UnicodeStringIO);
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/config/UnicodeStringEncodingTestCase.java
/* Copyright (C) 2008  Versant Inc.  http://www.db4o.com */

package com.db4o.db4ounit.common.config;

import com.db4o.internal.encoding.*;

/**
 * @exclude
 */
public class UnicodeStringEncodingTestCase extends StringEncodingTestCaseBase {

	protected Class stringIoClass() {
		return UnicodeStringIO.class;
	}

}
